config.md

```diff
@@ -346,3 +346,70 @@ Hover:
 
 A boolean that controls printing of desugared types, e.g:
 `vector<int>::value_type (aka int)`
+
+## Semantic Tokens
+{:.v17}
+
+Configure semantic tokens for code highlighting. Sample block (default):
+
+```yaml
+SemanticTokens:
+  DisabledKinds: []
+  DisabledModifiers: []
+```
+
+### DisabledKinds
+{:.v17}
+
+Disable a certain kinds of semantic tokens to be sent by clangd.
+
+The complete list of token kinds include:
+- Variable
+- LocalVariable
+- Parameter
+- Function
+- Method
+- StaticMethod
+- Field
+- StaticField
+- Class
+- Interface
+- Enum
+- EnumConstant
+- Typedef
+- Type
+- Unknown
+- Namespace
+- TemplateParameter
+- Concept
+- Primitive
+- Macro
+- Modifier
+- Operator
+- Bracket
+- InactiveCode
+
+### DisabledModifiers
+{:.v17}
+
+Disable a certain types of semantic token modifiers to be sent by clangd.
```

Comment:
suggest rewording as "Specify semantic token modifiers that clangd should not send"